guide/14-deep-learning/mistral_workflow_with_text_models.ipynb

```diff
@@ -0,0 +1,621 @@
+{
```

Comment:
<p>Once the data is prepared, <code>TextClassifier</code> model object can be instantiated as below with the following parameters:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1794/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>